[BUG]: small error when downloading subs only

### Program version

5.1.5

### Operating System

Linux

### Type

CLI

### Service

Crunchyroll

### Command used

aniDL --service crunchy -s GR24P4256 --dlsubs pt-BR --novids --noaudio --all 

### Show ID

GR24P4256

### Episode

all

### Console Output

```Shell
[ERROR] Found variable '${height}' in fileName but no values was internally found!
[ERROR] Found variable '${height}' in fileName but no values was internally found!
[ERROR] Found variable '${height}' in fileName but no values was internally found!
Subtitle downloaded: [CR] Hyouka - S01E03 [${height}p].03.por.Brazilian Portuguese.ass
```


### Additional Information

Subs are downloaded, but they names are [CR] Hyouka - S01E06 [${height}p].03.por.Brazilian Portuguese.ass. maybe add a default value for height will solve that.

Thankks for the script

--- a/modules/module.filename.ts
+++ b/modules/module.filename.ts
@@ -23,7 +23,10 @@ const parseFileName = (input: string, variables: Variable[], numbers: number, ov
   for (let i = 0; i < vars.length; i++) {
     const type = vars[i];
     const varName = type.slice(2, -1);
-    const use = overridenVars.find(a => a.name === varName);
+    let use = overridenVars.find(a => a.name === varName);
+    if (use === undefined && type === '${height}') {
+      use = { type: 'number', replaceWith: 0 } as Variable<string>;
+    }
     if (use === undefined) {
       console.info(`[ERROR] Found variable '${type}' in fileName but no values was internally found!`);
       continue;

